spec/features/users/user_signs_up_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+feature 'sign up' do
+
+  scenario 'Provided signup information is valid' do
+    visit root_path
+    click_link 'Sign Up'
+
+    fill_in 'First Name', with: 'Guy'
+    fill_in 'Last Name', with: 'Guyson'
+    fill_in 'Email', with: "dude@dude.com"
+    fill_in 'user_password', with: "password"
+    fill_in 'Password Confirmation', with: "password"
+    click_button 'Sign Up'
+
+    expect(page).to have_content("Sign up successful!")
+    expect(page).to have_content("Sign Out")
+  end
+
+
+  scenario 'Provided signup information is invalid' do
+    visit root_path
+    click_link 'Sign Up'
+    click_button 'Sign Up'
```

Comment:
This is confusing without the blank line, because it's unclear that you've navigated to the form without a closer look. (At first I thought you were clicking the Sign Up link twice and wondering why!) Just an example of how the blank lines clarify the flow of activity in a feature test.